build.gradle

```diff
@@ -0,0 +1,179 @@
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '5.0-SNAPSHOT-WILLH'
+        corda_gradle_plugins_version = '4.0.42'
+        kotlin_version = '1.2.71'
+        junit_version = '4.12'
+        quasar_version = '0.7.10'
+        slf4j_version = '1.7.25'
+        log4j_version = '2.11.2'
+        corda_platform_version = '4'
+        tokens_release_group='com.r3.corda.lib.tokens'
+        tokens_sdk_version='1.0'
+        confidential_id_release_group = "com.r3.corda.lib.ci"
+        confidential_id_release_version = "1.0"
+    }
+
+    repositories {
+        jcenter()
+        mavenCentral()
+        mavenLocal()
+        maven { url 'https://jitpack.io' }
+        maven { url "http://ci-artifactory.corda.r3cev.com/artifactory/corda-dev" }
```

Comment:
These should be `https` rather than `http` - it's a mistake that gets copied in most of our build gradles :) @florian-f pointed it out to me the other day!

Reply:
Done